src/components/NewGalaxyModal.js

```diff
@@ -0,0 +1,119 @@
+import _ from 'lodash/fp'
+import PropTypes from 'prop-types'
+import { Component, Fragment } from 'react'
+import { div, h, li, p, span, ul } from 'react-hyperscript-helpers'
+import { ButtonPrimary, Link } from 'src/components/common'
+import { withModalDrawer } from 'src/components/ModalDrawer'
+import TitleBar from 'src/components/TitleBar'
+import { machineTypes, profiles } from 'src/data/machines'
+import { normalizeRuntimeConfig } from 'src/libs/cluster-utils'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+
+
+const styles = {
+  row: {
+    display: 'flex',
+    alignItems: 'center',
+    marginTop: '1rem'
+  },
+  label: { fontWeight: 600, whiteSpace: 'pre' },
+  disabledInputs: {
+    border: `1px solid ${colors.dark(0.2)}`, borderRadius: 4, padding: '0.5rem'
+  },
+  whiteBoxContainer: { padding: '1rem', borderRadius: 3, backgroundColor: 'white' },
+  drawerContent: { display: 'flex', flexDirection: 'column', flex: 1, padding: '1.5rem' }
+}
+
+export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewClusterModal extends Component {
+  static propTypes = {
+    currentCluster: PropTypes.object,
```

Comment:
a lot of this is copied from the NewClusterModal so I need to clean this section up and remove some unused stuff